cypress/integration/e2e/table/update-workhours-info.spec.js

```diff
@@ -0,0 +1,285 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/// <reference types="cypress" />
+
+const nurseInitialWorkHours = { required: 136, actual: 164, overtime: 28 };
+const babysitterInitialWorkHours = { required: 80, actual: 108, overtime: 28 };
+
+const workerTypeToIdx = {
+  nurse: 0,
+  babysitter: 3,
+};
+
+function shiftCell(workerType, shiftIdx) {
+  return cy
+    .get(`[data-cy="${workerType}ShiftsTable"]`)
+    .children()
+    .children()
+    .eq(workerTypeToIdx[workerType])
+    .children()
+    .eq(shiftIdx)
+    .children()
+    .children();
+}
+
+const cellTypeToIdx = {
+  required: 0,
+  actual: 1,
+  overtime: 2,
+};
+
+function hoursInfoCell(workerType, cellType) {
+  return cy
+    .get(`[data-cy="${workerType}SummaryTable"]`)
+    .children()
+    .children()
+    .eq(workerTypeToIdx[workerType])
+    .children()
+    .eq(cellTypeToIdx[cellType]);
+}
+
+function testWorkHoursInfoUpdate(testData) {
+  const cellData = [testData.workerType, testData.shiftIndex];
+
+  if (testData.initialShiftCode) {
+    shiftCell(...cellData).contains(testData.initialShiftCode);
+    shiftCell(...cellData).click();
+  } else {
+    shiftCell(...cellData)
+      .parent()
+      .click();
+  }
+  cy.get('[data-cy="shiftDropdown"]').contains(testData.desiredShiftText).click({ force: true });
+
+  if (testData.desiredShiftCode) {
+    shiftCell(...cellData).contains(testData.desiredShiftCode);
+  } else {
+    shiftCell(...cellData).should("be.empty");
+  }
+
+  hoursInfoCell(testData.workerType, "required").contains(testData.expectedWorkHoursInfo.required);
+  hoursInfoCell(testData.workerType, "actual").contains(testData.expectedWorkHoursInfo.actual);
+  hoursInfoCell(testData.workerType, "overtime").contains(testData.expectedWorkHoursInfo.overtime);
+}
+
+context("Work hours info (summary table)", () => {
+  beforeEach(() => {
+    cy.visit(Cypress.env("baseUrl"));
+    cy.contains("Plik").click();
+    cy.get('[data-cy="file-input"]').attachFile("example.xlsx");
+  });
+
+  // sanity check in case schedule in the docs gets changed and these tests start failing because of it
+  it("Has expected initial values of workHourInfo in example schedule", () => {
```

Comment:
Bardzo słuszny test!
Mam też propozycję, żebyśmy ustawiali tutaj jakoś flagę która będzie sygnalizować czy odpalać dalej testy czy nie. 
Bo teraz nawet jeżeli ten test nie przejdzie, to nie wniesie to dużo informacji, bo nie przejdą też inne i będziemy mieli za dużo błędów.

Reply:
Nie udało mi się tego zrobić, na tyle na ile poszukałem o tym to sposobu żeby to zrobić po bożemu jeszcze nie dodali a workaroundów nie byłem w stanie zmusić do działania.

Reply:
Popatrzymy z @pectom 
Może nam uda się coś znaleźć